dataframe-jdbc/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/readJdbc.kt

```diff
@@ -322,19 +347,119 @@ public fun Connection.readDataFrame(
             limit,
             inferNullability,
             dbType,
+            strictValidation,
         )
 
         else -> throw IllegalArgumentException(
             "$sqlQueryOrTableName should be SQL query or name of one of the existing SQL tables!",
         )
     }
 
-/** SQL query is accepted only if it starts from SELECT */
-private fun isValid(sqlQuery: String): Boolean {
+/**
+ * Checks if a given string contains forbidden patterns or keywords.
+ * Logs a clear and friendly message if any forbidden pattern is found.
+ */
+private fun containsForbiddenPatterns(input: String): Boolean {
+    // List of forbidden patterns or commands
+    val forbiddenPatterns = listOf(
+        ";", // Separator for SQL statements
+        "--", // Single-line comments
+        "/*", // Start of multi-line comments
+        "*/", // End of multi-line comments
+        "DROP",
+        "DELETE",
+        "INSERT",
+        "UPDATE",
+        "EXEC",
+        "EXECUTE",
+        "CREATE",
+        "ALTER",
+        "GRANT",
+        "REVOKE",
+        "MERGE",
```

Comment:
forbidden statements are now still allowed if you write them in different capitalization